src/rewriter.ts

```diff
@@ -84,13 +84,19 @@ export abstract class Rewriter {
     this.writeNodeFrom(node, pos);
   }
 
-  writeNodeFrom(node: ts.Node, pos: number) {
+  writeNodeFrom(node: ts.Node, pos: number, end = Number.MAX_SAFE_INTEGER) {
```

Comment:
Can write:

writeNodeFrom(node: ts.Node, pos: number, end = node.getEnd()) {

and then drop the Math.min() below as well as calling node.getEnd().